.circleci/config.yml

```diff
@@ -64,32 +64,32 @@ jobs:
   build:
     <<: *executor
     steps:
-     - checkout
-     - run: |
-         set -x
-         export PULL_CACHE=yes
-         export AGENT_VERSION=latest
-         make image
-     - run: |
-         mkdir -p /tmp/workspace
-         docker save -o /tmp/workspace/image.tar quay.io/signalfx/signalfx-agent-dev:latest
-     - persist_to_workspace:
-         root: /tmp/workspace
-         paths:
-           - image.tar
+      - checkout
```

Comment:
This doesn't line up anymore

Reply:
I indented everything with an extra space because yamllint was complaining: `wrong indentation: expected 6 but found 5  (indentation)`